amm/interp/src/main/scala/ammonite/interp/AmmonitePlugin.scala

```diff
@@ -284,25 +284,30 @@ object LineNumberModifier {
                        topWrapperLen: => Int) = {
 
     object LineNumberCorrector extends g.Transformer {
+      import scala.reflect.internal.util._
+
+      private val trimmedSource = new BatchSourceFile(g.currentSource.file,
+        g.currentSource.content.drop(topWrapperLen))
```

Comment:
this is an optimisation to avoid re-trimming the source for every node on the tree